integration/testnet/network.go

```diff
@@ -699,13 +657,26 @@ func BootstrapNetwork(networkConf NetworkConfig, bootstrapDir string) (*flow.Blo
 		return nil, nil, nil, nil, err
 	}
 
+	numViewsInDKGPhase := uint64(100)
```

Comment:
need to make these configurable from localnet and testnet config, not hard-coded here